File: bfnlibrary/lib/data/dashboard_data.dart
class DashboardData {
  int invoices;
  int accounts, offers;
  String node;

  DashboardData(this.invoices, this.accounts, this.offers, this.node);

  DashboardData.fromJson(Map data) {
    this.invoices = data['invoices'];
    this.accounts = data['accounts'];
    this.offers = data['offers'];
    this.node = data['webAPIUrl'];
  }

  Map<String, dynamic> toJson() => <String, dynamic>{
        'offers': offers,
        'invoices': invoices,
        'accounts': accounts,
        'node': node,
      };
}

File: bfn-corda/clients/src/main/java/com/bfn/dto/DashboardData.java
package com.bfn.dto;

public class DashboardData {
    private int accounts, invoices, offers;
    private String node;

    public DashboardData(int accounts, int invoices, int offers, String node) {
        this.accounts = accounts;
        this.invoices = invoices;
        this.offers = offers;
        this.node = node;
    }

    public DashboardData() {
    }

    public int getAccounts() {
        return accounts;
    }

    public void setAccounts(int accounts) {
        this.accounts = accounts;
    }

    public int getInvoices() {
        return invoices;
    }

    public void setInvoices(int invoices) {
        this.invoices = invoices;
    }

    public int getOffers() {
        return offers;
    }

    public void setOffers(int offers) {
        this.offers = offers;
    }

    public String getNode() {
        return node;
    }

    public void setNode(String node) {
        this.node = node;
    }
}
